### PR TITLE
Support for LTI launches Fixes #1

### LINKED ISSUE #1: Integrate with PyLTI to Launch 1.3 from Canvas

### PR DESCRIPTION
**Pull request by jonespm**
I wrote up this guide in wiki to test here: https://github.com/tl-its-umich-edu/canvas-app-explorer/wiki/Configuring-LTI-1.3

Let me know if you have any feedback. This looks pretty good to me. We *might* need to switch to the json configuration before going to production for the cache to work correctly.

### CODE REVIEW

File: frontend/server.js [hunk 1]
```diff
@@ -2,13 +2,17 @@ const webpackDevServer = require('webpack-dev-server');
 const webpack = require('webpack');
 
 const config = require('./webpack.config.js');
+const publicPath = 'https://958ba9a1711d.ngrok.io/dist/'
```
Comment (zqian) [inline frontend/server.js:5]: Is this a placeholder for ngrok url?

File: backend/settings.py [hunk 2]
```diff
@@ -134,4 +135,27 @@
...
+
+# TODO: Switch this to CSP for additional security
+X_FRAME_OPTIONS = 'ALLOWALL'
+
+# So request works over the proxy
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+
+DB_CACHE_CONFIGS = os.getenv('DB_CACHE_CONFIGS',
+                           {'CACHE_TTL': 600, 'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
+                            'LOCATION': 'django_myla_cache',
```
Comment (zqian) [inline backend/settings.py:148]: still says django_myla_cache here

File: backend/settings.py [hunk 3]
```diff
@@ -134,4 +135,27 @@
...
+                            'CACHE_KEY_PREFIX': 'myla',
```
Comment (zqian) [inline backend/settings.py:149]: and prefix is still myla

**Review by zqian — COMMENTED** [review]
I have tested the code and made some minor changes to the installation wiki page

File: backend/settings.py (hunk 2 above)
Comment (jonespm) [reply backend/settings.py:148]: Thanks! I copied some of this from myla, I'll update this name.

File: frontend/server.js (hunk 1 above)
Comment (jonespm) [reply frontend/server.js:5]: Hmm, I think I just left that in there and it should use the configured value. I'll remove it. Good catch!

Comment (jonespm) [reply frontend/server.js]: I'm trying to figure out why I had this in there in the first place. There were some issues getting this to work with ngrok. 

This configuration and frontend server is only used in local development, it should be compiled down to one server (just the backend) in production.

**Review by zqian — APPROVED** [review]

File: frontend/server.js (hunk 1 above)
Comment (jonespm) [reply frontend/server.js]: I think I remember why I had this now. The backend code actually uses some of these variables from the frontend to figure out how to serve up the content. I'd temporarily hardcoded this but figured it out more and fixed it.

**Comment by jonespm** [comment]
I'd like to fix #80 before this one, otherwise this causes warnings after running this migration that there are more migrations to run. I should have that fixed tomorrow and we can get these both in.

**Pull request merged**